sdk/Sdk.Generators/Targets/Microsoft.Azure.Functions.Worker.Sdk.Generators.props

```diff
@@ -0,0 +1,5 @@
+﻿<Project>
+  <ItemGroup>
+    <CompilerVisibleProperty Include="FunctionsEnablePlaceholder" />
```

Comment:
Need to sync with @satvu to understand whether we shouldFunctionsMetadataSourceGen_Enabled here.

Reply:
For reference, `FunctionsMetadataSourceGen_Enabled` was added as a "CompilerVisibleProperty" in the [meta SDK package](https://github.com/Azure/azure-functions-dotnet-worker/blob/ae0cbe0f08b2ee7418fc6ded8d020b5b4b9bdfb2/sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.props) but we should probably pick just one location to keep these all together.